examples/full-example.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "My first Module",
+    "archiveName": "mycompany/my-first-module",
+    "sourceDir": "new_files",
+    "version": "auto",
+
+    "shortDescription": "",
+    "description": "",
+
+    "developer": "Robin Wieschendorf",
+    "developerWebsite": "https://robinwieschendorf.de",
+    "website": "https://github.com/RobinTheHood/modified-std-module",
+
+    "category": "productivity",
+    "price": "29.90",
+    "tags": "verstand, shipping, paket",
```

Comment:
I hope you have a lot of "verstand" 😉

Reply:
I figured it was a typo but copied it from the documentation for completeness :D

https://github.com/RobinTheHood/module-loader.de/blob/1a60419342b082f800e90b4f361f1162110859a6/public/docs/moduleinfo.php#L224

Reply:
Oh shit. Maybe you can fix it for me 😅 Thx